diagnostic.md

```diff
@@ -6,49 +6,53 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+One of the purpose of backend is to store data that can be accessed, manipulated and deleted. To be able to do that we would need front end to interact with backend.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+Route takes request and gives it to controller
```

Comment:
This is true, but it doesn't answer the question "Which layer in the MVC pattern is used by the controller to fetch data?" The model.